apps/demo-ngrx-classic/src/app/app.component.e2e.ts

```
@@ -0,0 +1,85 @@
+  }
+
+  static navigationEndGuard(event: unknown): event is NavigationEnd {
+    return event instanceof NavigationEnd;
+  }
+
+  ngOnInit(): void {
+    this.activeLink = this.router.events.pipe(
+      filter(AppComponent.navigationEndGuard),
+      map(AppComponent.navigationEndRoute),
+    );
+    const context = this;
```

Biome (1.9.4)

[error] This aliasing of this is unnecessary.

Arrow functions inherits `this` from their enclosing scope.
Safe fix: Use this instead of an alias.


(lint/complexity/noUselessThisAlias)